### ISSUE TITLE
Path replacement rule not working as expected

### ISSUE THREAD

**Issue by dedalusj** [containous/traefik#1775]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Run the following docker compose file

```yml
    version: '3'
    services:
      traefik:
        image: traefik:1.3.1
        command: --web --docker --docker.domain=docker.localhost --logLevel=DEBUG
        networks:
          - web
        ports:
          - "80:80"
          - "8080:8080"
        volumes:
          - /var/run/docker.sock:/var/run/docker.sock
          - /dev/null:/traefik.toml
      whoami:
        image: emilevauge/whoami
        networks:
          - web
        labels:
          - "traefik.backend=whoami"
          - "traefik.app.frontend.rule=PathPrefix:/whoami"
          - "traefik.healtcheck.frontend.rule=Path:/whoami/healthcheck;ReplacePath:/foo"
    networks:
      web:
        driver: bridge
```
and navigate to `http://localhost/whoami/healthcheck` in the browser.

### What did you expect to see?

whoami should report a request to the `/foo` path.

### What did you see instead?

```
Hostname: d24d01ae0a01
IP: 127.0.0.1
IP: 172.19.0.2
GET /whoami/healthcheck HTTP/1.1
Host: localhost
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
Accept-Encoding: gzip, deflate, sdch, br
Accept-Language: en-GB,en-US;q=0.8,en;q=0.6
Cache-Control: max-age=0
Upgrade-Insecure-Requests: 1
X-Forwarded-For: 172.19.0.1
X-Forwarded-Host: localhost
X-Forwarded-Proto: http
X-Forwarded-Server: 759fcea11e1b
X-Replaced-Path: /whoami/healthcheck
```

The presence of `X-Replaced-Path` in the header shows the `ReplacePath` rule was executed but no path substitution happened.

### Output of `traefik version`:

```
Version:      v1.3.1
Codename:     raclette
Go version:   go1.8.3
Built:        2017-06-16_11:21:48AM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

See docker compose file above

**Comment by ldez** [comment]
Closed by #1859.

**Comment by ldez** [comment]
Closed by #1859.

**Comment by ldez** [comment]
Closed by #1859.
